drivers/ZW_Dimmer_Rotary/driver.settings.compose.json

```diff
@@ -0,0 +1,58 @@
+[
+  {
+    "type": "group",
+    "label": {
+      "en": "Button behaviour",
+      "nl": "Gedrag knoppen"
+    },
+    "collapsed": false,
+    "children": [
+      {
+        "$extends": "externalSwitchInclusion",
+        "$id": "external_switch_inclusion"
+      }
+    ]
+  },
+  {
+    "type": "group",
+    "label": {
+      "en": "Dimming",
+      "nl": "Dimmen"
+    },
+    "collapsed": true,
+    "children": [
+      {
+        "$extends": "dimDuration",
+        "$id": "dim_duration"
+      },
+      {
+        "$extends": "minimumBrightness",
+        "$id": "minimum_brightness"
+      }
+    ]
+  },
+  {
+    "type": "group",
+    "label": {
+      "en": "Other",
+      "nl": "Overige"
+    },
+    "collapsed": true,
+    "children": [
+      {
+        "$extends": "saveState",
+        "$id": "save_state"
+      },
+      {
+        "$extends": "sendBasicReport",
+        "$id": "send_basic_report"
+      },
+
+      {
```

Comment:
The rotary dimmer does not support driving type (you cannot configure leading edge (GLI?) vs. falling edge trigger (GLE?)). The rotary dimmer is GLE only.